### DISCUSSION TITLE
Minor bug in jsPsych's WebGazer fork

### DISCUSSION THREAD

**Discussion by bbonf** [jspsych/jsPsych#2560] [General]
Hello,

I'm posting this here because jsPsych/WebGazer doesn't have an issue tracker.

The WebGazer fork contains a small bug in the prediction pipeline:
https://github.com/jspsych/WebGazer/blob/master/src/util.mjs#L27
It has already been fixed upstream some time ago:
https://github.com/brownhci/WebGazer/blob/master/src/util.mjs#L34

The jsPsych version resizes the eye fragments of the video stream into 6x6 images instead of the intended 10x6.
I only tested it quite casually so far, but it seems like the difference in accuracy this produces is not obvious.
Fixing it however does come with a certain performance cost, as there's more data to process, and I can notice that difference in the time resolution of collected data.

I'm not really sure what should be done here, but after spending some time puzzling about it, I thought I might share it here.

**Comment by becky-gilbert** [comment]
Thanks for flagging this @bbonf! I've created a new issue to track this - #2561.
